diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/WhiteSpaceRule.kt

```diff
@@ -430,6 +458,7 @@ class WhiteSpaceRule(configRules: List<RulesConfig>) : DiktatRule(
 
     companion object {
         private val log = LoggerFactory.getLogger(CompactInitialization::class.java)
+        private const val MAX_LENGTH = 120L
```

Comment:
I don't see that this property is in used. But if it is, can we move it to property file instead of constant in source code?

Reply:
Yes, I deleted the piece of code where it was used, thanks for the comment [f01b794](https://github.com/saveourtool/diktat/pull/1373/commits/f01b794c2b2b112dd192cdbc2571b3aa1aae5141)